tests/test_magnification_dialog.py

```diff
@@ -40,6 +43,7 @@ def test_magnification_ui(
     back_fiducial1,
     back_fiducial2,
     expected_magnification_params,
+    double_click,
```

Comment:
```suggestion
    click_twice,
```
Again, I think "click twice" is a bit more accurate than double-clicking. It's not a double-click mouse action, right?